.github/workflows/ci.yml

```
@@ -0,0 +1,120 @@
+    - name: Run linting
+      working-directory: ./MesaChain-BE
+      run: pnpm run lint
+    
+    - name: Run tests
+      working-directory: ./MesaChain-BE
+      run: pnpm run test:cov
+      env:
+        DATABASE_URL: postgresql://test_user:test_password@localhost:5432/test_db
+    
+    - name: Upload coverage reports
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)